logic/broadcaster.js

```diff
@@ -138,10 +148,12 @@ Broadcaster.prototype.getPeers = function(params, cb) {
 };
 
 /**
- * Adds new object {params, options} to queue array .
- * @param {Object} params
- * @param {Object} options
- * @return {Object[]} queue private variable with new data
+ * Adds new object {params, options} to queue array.
+ *
+ * @param {Object} params - Description of the param
+ * @param {Object} options - Description of the param
+ * @returns {Object[]} queue private variable with new data
```

Comment:
Good question, both syntax are valid, see: http://usejsdoc.org/tags-type.html
Do you have a preference? 
If `Object[]` is only used here, I would change it to `Array.<Object>`, but I don't have a strong opinion on it.

Reply:
I'd be tempted to get rid of nesting altogether and just use `Array` (and revisit later). But if this is the only place we use `[]` it would be relatively easy to just switch to `<>`.

Reply:
As discussed, we will open a new issue for standardizing typedefs in all files